examples/06_productivity_boosters/01_namedArguments.md

```diff
@@ -1,5 +1,7 @@
 # Named Arguments
 
+As most of programming languages (Java, C++, and other), Kotlin supports passing arguments to methods and constructors according to their order of definition.
+Kotlin also supports [named arguments](https://kotlinlang.org/docs/reference/functions.html#named-arguments) to allow clearer invocations and avoid mistakes in arguments order. Such mistakes are hard to find because they are not detected by the compiler, for example, if two sequential arguments have the same type.
```

Comment:
Kotlin also supports [named arguments](https://kotlinlang.org/docs/reference/functions.html#named-arguments) to allow clearer invocations and avoid mistakes with the order of arguments. Such mistakes are hard to find because they are not detected by the compiler, for example, when two sequential arguments have the same type.